Solutions/Proofpoint On demand(POD) Email Security/Parsers/ProofpointPOD.yaml

```diff
@@ -314,4 +379,245 @@ FunctionQuery: |
+    	        	MsgNormalizedHeaderReturnPathHashed= tostring(parse_json(msg).normalizedHeader.["return-pathHashed"]),
+    	        	MsgNormalizedHeaderXMailer= tostring(parse_json(msg).normalizedHeader.["x-mailer"]),
+    		        MsgNormalizedHeaderReturnPath= tostring(parse_json(msg).normalizedHeader.["return-path"]),
+    		        MsgHeaderXMailer= tostring(parse_json(msg).header.["x-mailer"]),
+    		        MsgHeaderReturnPathHashed= tostring(parse_json(msg).header.["return-pathHashed"]),
+    		        MsgHeaderReturnPath= tostring(parse_json(msg).header.["return-path"]),
+    		        FilterModulesSpamCharsets= tostring(parse_json(filter).modules.spam.charsets),
+    		        FilterModulesDmarcAlignment= tostring(parse_json(filter).modules.dmarc.["alignment"]),
+    		        MsgNormalizedHeaderXOriginatingIp= tostring(parse_json(msg).normalizedHeader.["x-originating-ip"]),
+    		        MsgHeaderXOriginatingIp= tostring(parse_json(msg).header.["x-originating-ip"]),
+    		        MsgNormalizedHeaderReplyTo= tostring(parse_json(msg).normalizedHeader.["reply-to"]),
+    		        MsgNormalizedHeaderReplyToHashed= tostring(parse_json(msg).normalizedHeader.["reply-toHashed"]),
+    		        MsgHeaderReplyToHashed= tostring(parse_json(msg).header.["reply-toHashed"]),
+    		        MsgHeaderReplyTo= tostring(parse_json(msg).header.["reply-to"]),
+    		        FilterModulesUrldefenseCountsNoRewriteIsEmail= toreal(parse_json(filter).modules.urldefense.counts.noRewriteIsEmail),
+    		        FilterModulesPdrV2Rscore= toreal(parse_json(filter).modules.pdr.v2.rscore),
+    		        FilterOrigGuid= tostring(parse_json(filter).origGuid),
+    		        MsgNormalizedHeaderCc= tostring(parse_json(msg).normalizedHeader.["cc"]),
+    		        MsgNormalizedHeaderCcHashed=tostring(parse_json(msg).normalizedHeader.["ccHashed"]),
+    		        MsgParsedAddressesCcHashed=tostring(parse_json(msg).parsedAddresses.["ccHashed"]),
+    		        MsgParsedAddressesCc= tostring(parse_json(msg).parsedAddresses.["cc"]),
+    		        MsgHeaderCcHashed= tostring(parse_json(msg).header.["ccHashed"]),
+    		        MsgHeaderCc= tostring(parse_json(msg).header.["cc"]),
+    		        FilterModulesAvVirusNames= tostring(parse_json(filter).modules.av.virusNames),
+    		        FilterThrottleIp=tostring(parse_json(filter).throttleIp),
+    		        FilterModulesUrldefenseCountsNoRewriteIsUnsupportedScheme= toreal(parse_json(filter).modules.urldefense.counts.noRewriteIsUnsupportedScheme),
+    		        FilterModulesUrldefenseCountsNoRewriteIsSchemeless= toreal(parse_json(filter).modules.urldefense.counts.noRewriteIsSchemeless),
+    		        FilterModulesUrldefenseCountsNoRewriteIsMaxLengthExceeded= tostring(parse_json(filter).modules.urldefense.counts.noRewriteIsMaxLengthExceeded),
+    		        FilterModulesUrldefenseCountsNoRewriteIsExcludedDomain= toreal(parse_json(filter).modules.urldefense.counts.noRewriteIsExcludedDomain),
```

Comment:
Why such big column names?

Reply:
These columns are same as v1, so not changing them but exacting the same columns as of v1 parser.